news/2021-12-16-project-loved-december-2021.md

```diff
@@ -0,0 +1,523 @@
+---
+layout: post
+title: "Project Loved: December 2021"
+date: 2021-12-16 12:00:00 +0000
```

Comment:
yeah that date sure passed

Reply:
I'd prefer if Eph takes care of this cause last time I did that, it fucked everything up, lol...